gulpfile.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+var gulp = require('gulp');
+var del = require('del');
+var eslint = require('gulp-eslint');
+var gutil = require('gulp-util');
```

Comment:
__[eslint]__
*  Error - gutil is defined but never used (no-unused-vars)

*  Error - Strings must use doublequote. (quotes)